jubako/src/main/java/com/justeat/jubako/ContentAdapterContentDescriptionCollectionListener.kt

```diff
@@ -10,8 +10,8 @@ class ContentAdapterContentDescriptionCollectionListener(
     }
 
     override fun notifyItemInserted(index: Int) {
-        logger.log(TAG, "Notify Item Inserted", "index: $index")
-        contentAdapter.notifyItemInserted(index)
+        // logger.log(TAG, "Notify Item Inserted", "index: $index")
```

Comment:
Intentionally commented out ? Should you just remove ?

Reply:
@Electryc thanks man yeh I think all these notify functions are now redundant now progress loading is all handled by `ProgressAdapter`